### DISCUSSION TITLE
FilePicker Throw Exceptions with Net  7

### DISCUSSION THREAD

**Discussion by sturlath** [Megabit/Blazorise#4394] [Q&A]
Is this this an issue with Blazorise @David-Moreira ? I’m guessing so since its a .net core issue (see link in the question below).

https://support.abp.io/QA/Questions/4127/FileEdit-and-FilePicker-Throw-Exceptions-with-Net-7-and-ABP-700-rc2

I just wanted to share this with you in case there was somehting you could do about it (maybe this has been reported here before, is so sorry for the spam)

**Comment by David-Moreira** [comment]
Right seems like it, I personally didn't find that error on my tests. Have you gone through the whole issue? There's some comments about making sure the protocol is web sockets instead of long polling for example, Have you tried that?

**Reply by sturlath** [reply]
No I haven’t since I can’t use anything of your new stuff until A release of abp 7.0 😞.. 27. Des is the release..

Rest assured that I’ll be the first person to report issues if any 😅
